test/integration/answer/AnswerIT.java

```diff
@@ -169,4 +167,36 @@ public void writingInAReadTransactionThrows() {
             }
         }
     }
+
+    @Test
```

Comment:
need tests for the other option too - `explain`

Reply:
Have added